### PR TITLE
⚠️ Build `master` tag on master instead of latest

### PR DESCRIPTION
**Pull request by vincepri**
Signed-off-by: Vince Prignano <vincepri@vmware.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Related to #1510

### CODE REVIEW

**Comment by chuckha** [comment]
Are there any docs that should be updated to reflect this change?

File: Makefile [hunk 1]
```diff
@@ -246,7 +246,7 @@ release-binary: $(RELEASE_DIR)
 release-staging: ## Builds and push container images to the staging bucket.
 	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-build-all docker-push-all release-alias-tag
 
-RELEASE_ALIAS_TAG=$(shell if [ "$(PULL_BASE_REF)" = "master" ]; then echo "latest"; else echo "$(PULL_BASE_REF)"; fi)
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
```
Comment (ncdc) [inline Makefile]: You can probably get rid of this and use PULL_BASE_REF below on line 253?

**Comment by ncdc** [comment]
Good point. `tip` is also a possibility

**Pull request merged**